presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -54,7 +54,7 @@ HDFS Configuration
 
 Presto configures the HDFS client automatically for most setups and
 does not require any configuration files. In some rare cases, such
-as when using federated HDFS, it may be necessary to specify additional
+as when using federated HDFS or high availability Hive, it may be necessary to specify additional
```

Comment:
This paragraph is talking about when it is necessary to provide Hadoop config files. I'm assuming "high availability Hive" means multiple metastores, but custom Hadoop config files are not necessary for that.